client/src/components/Form/Form.scss

```
@@ -131,6 +131,14 @@
   &:empty {
     display: none;
   }
+
+  .icon {
+    &::before {
+      vertical-align: middle;
+      margin-right: 6px;
+      font-size: 16px;
```

Is there a variable we can use here instead of hardcoding this size?

---

Perhaps:

```suggestion
      font-size: $font-size-lg;
```

@sachajudd any ideas?

---

I thought we'd run into trouble with using this variable (as it calculates out to `16.003px`) but I've just tested it again and it seems fine, so yep apply it 👍